float.ml

```diff
@@ -569,11 +598,15 @@ let set_neg a l u =
   a.(1) <- -. l;
   a.(2) <- u
 
-(** [set_neg_lower a f] sets lower bound of positive normalish to [-. f] *)
+(** [set_opp_pos_lower a f] sets lower bound of positive normalish to [-. f] *)
```

Comment:
No. when the interval is [4.0…5.0], the bounds are 4.0 and 5.0. The fact that one is stored as its opposite doesn't change this fact.